src/test/java/org/takes/rs/RsPrintTest.java

```diff
@@ -70,6 +75,32 @@ public void flushBodyEvenWhenExceptionHappens() throws IOException {
         );
     }
 
+    /**
+     * RsPrint can close body contents even when exception happens.
+     * @throws IOException If some problem inside
+     */
+    @Test
+    public void closeBodyEvenWhenExceptionHappens() throws IOException {
+        final IOException exception = new IOException("Smth went wrong");
+        final FailOutputStream output = new FailOutputStream(exception);
+        final FakeInput input = new FakeInput(new InputStreamOf("abc"));
+        try {
+            new RsPrint(new RsText(input))
+                .printBody(output);
+        } catch (final IOException ex) {
+            MatcherAssert.assertThat(
```

Comment:
@olenagerasimova What are we asserting with this? We should have just one assertion per test, so please remove it and create another test method for it if this assertion is relevant.